packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -83,11 +81,15 @@ export const Field: FunctionComponent<FieldProps> & Statics = withTheme(
       } = props;
 
       const TagLabel = as === "fieldset" ? "legend" : "label";
+      const componentClass = useCx(
+        styles.root,
+        as === "fieldset" && styles.rootFieldset
```

Comment:
consider renaming this class to `rootFieldset`. I don't love the convention because `root` is about the only class I want to couple to its location in the DOM.

Reply:
fixed